.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+ci:
+    autoupdate_schedule: 'monthly'
+    autofix_prs: false
+
+repos:
+-   repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
+      - id: black-jupyter
```

Comment:
I think `black-jupyter` supersedes `black`. See also: https://github.com/projectmesa/mesa/blob/34eb008aa849c70f80e0fa69198d82ba2bf82193/.pre-commit-config.yaml#L9

Reply:
Thanks for the suggestions; incorporated!

Reply:
This comment it out of date -- something is wrong with Github and it is not registering correctly. 
@catherinedevlin has fixed the issue -- it is identifiable in the file. This is super weird behavior on github ... took us awhile to troubleshoot.